app/client/src/tests/unit/!Banner.test.js

```diff
@@ -0,0 +1,5 @@
+test(`Write the banner`, () => {
+    console.log(`##################################################
+#############  BEGIN UNIT TESTING HERE  ###############
+##################################################`);
+});
```

Comment:
Try to make sure you add new lines at the end of your files. Why we do this: [https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

Reply:
Will do!

Reply:
❤️